### DISCUSSION TITLE
Access partial content downloaded by cancelled request

### DISCUSSION THREAD

**Discussion by andrisi** [sindresorhus/got#1665] [Q&A] [answered]
#### What problem are you trying to solve?

Sometimes I need to cancel a request, but also access partial content downloaded by it. I cancel it from the progress event.

#### Describe the feature

I see request.isCanceled is useful, but is a rejected promise and properties don't seem to be accessible. Thanks.

#### Checklist

- [x] I have read the documentation and made sure this feature doesn't already exist.

**Comment by szmarczak** [comment, marked as answer]
Use Stream API.

**Reply by andrisi** [reply]
Thanks @szmarczak will do so.
